mockredis/client.py

```diff
@@ -301,11 +302,32 @@ def _rename(self, old_key, new_key, nx=False):
     def dbsize(self):
         return len(self.redis.keys())
 
-    # String Functions #
+    def _decode(self, value):
```

Comment:
Having a decorator for decoding return values based on decode_responses could have been a cool solution maybe.. just a thought :) it's good as it is too